src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelMetadata.cs

```diff
@@ -238,8 +252,8 @@ public string PropertyName
         protected IModelMetadataProvider Provider { get; set; }
 
         /// <returns>
-        /// Gets <c>T</c> if <see cref="ModelType"/> is <see cref="Nullable{T}"/>;
-        /// <see cref="ModelType"/> otherwise.
+        /// Gets runtime <see cref="Type"/> of <see cref="Model"/> if <see cref="Model"/> is non-<c>null</c> and
+        /// <see cref="ModelType"/> is not <see cref="Nullable{T}"/>; <see cref="ModelType"/> otherwise.
```

Comment:
@rynowak this was one of the more misleading doc comments we discussed recently...